### PR TITLE
fixed the warnings, HAPPY NOW!

### PR DESCRIPTION
**Pull request by GeneDrive**

### CODE REVIEW

File: server/Migrations/20221222215900_done.cs [hunk 1]
```diff
@@ -0,0 +1,387 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```
Comment (ManoloTonto1) [inline server/Migrations/20221222215900_done.cs:1]: Add the whole migrations folder to the gitignore pls @GeneDrive

**Review by ManoloTonto1 — CHANGES_REQUESTED** [review]
just add the migrations folder to the gitignore inside the server folder

File: server/Migrations/20221222215900_done.cs (hunk 1 above)
Comment (GeneDrive) [reply server/Migrations/20221222215900_done.cs:1]: reeeeeee

Comment (GeneDrive) [reply server/Migrations/20221222215900_done.cs:1]: I did just now

Comment (ManoloTonto1) [reply server/Migrations/20221222215900_done.cs:1]: This file shouldn't exist after the gitignore thing. Find a way to ignore it

File: server/Migrations/theaterContextModelSnapshot.cs [hunk 2]
```diff
@@ -22,7 +22,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .ValueGeneratedOnAdd()
```
Comment (ManoloTonto1) [inline server/Migrations/theaterContextModelSnapshot.cs:22]: This file should also not exist

**Review by ManoloTonto1 — APPROVED** [review]
🚀 Merge it!

**Pull request merged**